orgs/beta.json

```diff
@@ -1,6 +1,6 @@
 {
   "orgName": "EDA - Beta Test Org",
-  "edition": "Developer",
+  "edition": "Enterprise",
```

Comment:
I also updated the beta and release org configurations to be Enterprise orgs, per @jstvz suggestion.